superduperdb/ext/openai/model.py

```diff
@@ -46,8 +54,8 @@ def __post_init__(self):
             msg = f'model {self.model} not in OpenAI available models, {mo}'
             raise ValueError(msg)
 
-        self.syncClient = SyncOpenAI()
-        self.asyncClient = AsyncOpenAI()
+        self.syncClient = SyncOpenAI(**self.client_kwargs)
```

Comment:
Are synchronous and asynchronous parameter definitions the same? If they are the same, they can be combined into one parameter?